packages/components/pie-webc-testing/src/helpers/page-object/base-page.ts

```diff
@@ -1,10 +1,86 @@
 import { type Page } from '@playwright/test';
+import { buildUrl } from './storybook-extensions';
+
+declare global {
+    interface Window { __eventsArray: Array<string>; }
```

Comment:
I think this is great for now but it might be worth thinking about how we want to handle multiple components pushing events at the same time, if we ever want to do that.